docs/en/02_Developer_Guides/14_Files/02_Images.md

```diff
@@ -200,13 +200,112 @@ SilverStripe\Core\Injector\Injector:
       Quality: 90
 ```
 
-### Lazy Loading
+### Lazy Loading {#lazy-loading}
 
-Images added via the WYSIWYG editor and as objects on templates are [lazy loaded](https://developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading) by default.
+Most modern browsers support the ability to "lazy load" images by adding a `loading="lazy"` attribute 
+to the `<img />` tag. This defers the loading of images not in the viewport to improve the initial 
+page load performance.
 
-To disable lazy loading for an individual image in a template, use `$MyImage.LazyLoad(false)`.
+Silverstripe CMS automatically adds the `loading="lazy"` to images added in an HTML editor field
+and to images rendered via a SS template file.
 
-To out of lazy loading globally, notably if you already have a custom lazy loading implementation, use the following yml config:
+Read [Browser-level image lazy-loading for the web](https://web.dev/browser-level-image-lazy-loading/)
+on _web.dev_ for more information.
+
+#### Selectively disabling lazy loading
+
+Images that are expected to be initially visible on page load, should be _eager_ loaded. This 
+provides a small performance gain since the browser doesn't have to render the entire page layout 
+before determining if the images need to be loaded. When in doubt, it's usually preferable to lazy 
+load the image.
+
+Images that don't have dimensions should not be lazy loaded as that might alter the layout of the 
+page after the initial page load. 
+
+Content authors have the ability to selectively disable lazy loading when inserting images in an 
+HTML editor field.
+
+Developers can selectively disable lazy loading for individual image in a SS template by calling 
+`LazyLoad(false)` on the image variable. Developers can allow content authors to control the 
+loading attribute of a specific image by adding lazy load field next to the 
+[`UploadField`](api:SilverStripe\Assets\UploadField).
+
+```php
+<?php
```

Comment:
Tested this locally and it works fine.

Reply:
I've reorganised those examples a bit:
- I've split the SS template code sample in two and moved the Image example above the PHP example.
- I've renamed the follow up section "Controlling lazy loading in for `<img>` tags in SS templates" and adjusted the content accordingly
- The bit about content authors selectively disabling lazy loading in WYSYWIG has been moved to the intro.

Let me know if that makes more sense?